### DISCUSSION TITLE
como subir proyecto aqui

### DISCUSSION THREAD

**Discussion by Eliezer-2626** [community/community#146182] [New to GitHub]
buenas tardes , como subo un proyecto y  luego como me doy cuenta que esta reddy , no conozco nada y estoy incursionando en este mundo de la programación 
/

**Comment by NickOsipov** [comment]
@Eliezer-2626 Hola! 

Te ayudo a explicar cómo subir tu proyecto a GitHub:

1. Para subir tu proyecto, sigue estos pasos básicos:
   - Entra a [GitHub Repositories](https://github.com/Eliezer-2626?tab=repositories) y haz clic en el botón verde "New" para crear un nuevo repositorio
   - Dale un nombre a tu repositorio
   - Puedes añadir una descripción opcional
   - Elige si quieres que sea público o privado

2. Después, desde tu computadora necesitarás:
   - Instalar Git en tu computadora
   - Abrir la terminal o línea de comandos
   - Navegar hasta la carpeta de tu proyecto
   - Ejecutar estos comandos básicos:
```bash
git init
git add .
git commit -m "initial commit"
git branch -M main
git remote add origin URL-DE-TU-REPOSITORIO
git push -u origin main
```

3. Para verificar que todo está listo ("ready"):
   - Visita la URL de tu repositorio en GitHub
   - Deberías ver todos tus archivos ahí
   - En la pestaña "Commits" podrás ver los cambios que has subido
   - La marca verde ✓ indica que todo se subió correctamente

**Comment by ghostinhershell** [comment]
Hi @Eliezer-2626, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
